web/src/app/schedules/ScheduleRuleList.js

```diff
@@ -13,9 +12,17 @@ import ScheduleRuleCreateDialog from './ScheduleRuleCreateDialog'
 import { ruleSummary } from './util'
 import ScheduleRuleEditDialog from './ScheduleRuleEditDialog'
 import ScheduleRuleDeleteDialog from './ScheduleRuleDeleteDialog'
-import { useResetURLParams, useURLParam } from '../actions'
 import { GenericError } from '../error-pages'
 import Spinner from '../loading/components/Spinner'
+import { DateTime } from 'luxon'
+import makeStyles from '@mui/styles/makeStyles'
+import { useScheduleTZ } from './useScheduleTZ'
+
+const useStyles = makeStyles({
+  popper: {
+    opacity: 1,
+  },
+})
```

Comment:
Not sure if we need this, I removed it and didn't see a difference in styling locally